src/v/cluster/members_manager.cc

```diff
@@ -153,9 +153,10 @@ ss::future<> members_manager::handle_raft0_cfg_update(
           auto diff = calculate_brokers_diff(_members_table.local(), cfg);
           auto added_brokers = diff.additions;
           return _members_table
-            .invoke_on_all([cfg = std::move(cfg)](members_table& m) mutable {
-                m.update_brokers(cfg.brokers());
-            })
+            .invoke_on_all(
```

Comment:
would it be possible to split changes in `cluster` submodule from changes in `admin_server` into separate commits ?

Reply:
I agree, along with a commit that explains the change in `cluster::` (e.g. `update_offset`)

Reply:
done